src/types.rs

```diff
@@ -8,6 +8,10 @@ pub enum Error<E> {
     Comm(E),
     /// Invalid input data provided
     InvalidInputData,
+    /// Invalid output time provided h,m,s
```

Comment:
```suggestion
    /// Invalid time set in device: (hours, minutes, seconds)
```
This is returned when the device has an incorrect time set, right? Could you change the documentation to something like the above? (also the same for the Date variant)

Reply:
Yes that's correct. When a non-battery enabled device first starts up, you could interrogate the time before it has been set, but it will return zeros which is out of range for the `NaiveDate` or `NaiveTime`. Rather than panicing, we return the invalid values.